hpoflow/optuna_transformers.py

```diff
@@ -26,35 +37,25 @@ def __init__(
         log_training_args: bool = True,
         log_model_config: bool = True,
     ):
-        """
-        Check integration package dependencies and initialize class.
-
-        Args:
-            trial: OptunaMLflow object
-            log_training_args: Whether to log all Transformers TrainingArguments as MLflow params
-            log_model_config: Whether to log the Transformers model config as MLflow params
-        """
-        self._initialized = False
-        self._log_artifacts = False
-        self._ml_flow = mlflow
```

Comment:
@twolffpiggott just fyi: I do not want to add docstrings to `__init__`. Just add it to the class documentation.